07-space.Rmd

```diff
@@ -1,31 +1,29 @@
 # Spatial data {#space}
 
-From displaying simple point data to examining collision density along routes or between areas, the geographic property of STATS19 data is one of its most useful attributes. Mapping is a hugely useful and powerful aspect of R and has many applications in road safety, both in understanding geographic trends and presenting insight to colleagues. This aspect of R is covered in detail in the book [Geocomputation With R](https://geocompr.robinlovelace.net/index.html) [@lovelace_geocomputation_2019]. By mapping collision data in R, you can add layers containing other geo-coded datasets to further understand the reasons for certain trends. This can lead to new opportunities for intervention and collaboration with other parties who may have mutually compatible solutuions for reaching their goals.
```

Comment:
Did you mean to delete this new intro @PublicHealthDataGeek ?

Reply:
No I dont think I saw that.  Did you add it when I was commenting?  I might of accidentally deleted it as sometimes I seem to accdientally select stuff on my new laptop.  I think this is a useful intro - one of the general points I think is making sure the sections link back to why this skill/bit of code/function in R is useful to road safety research - I think I have added stuff in to help with this but it may be worth doing an overall review to make sure it links to this aim.

Reply:
> No I dont think I saw that. Did you add it when I was commenting?

Likely, I'm happy to re-add those bits, no worries.